contracts/libraries/SystemContractHelper.sol

```diff
@@ -270,6 +270,8 @@ library SystemContractHelper {
     function getZkSyncMeta() internal view returns (ZkSyncMeta memory meta) {
         uint256 metaPacked = getZkSyncMetaBytes();
         meta.gasPerPubdataByte = getGasPerPubdataByteFromMeta(metaPacked);
+        meta.heapSize = getHeapSizeFromMeta(metaPacked);
```

Comment:
I think that the constants for getting heap size and aux heap size are incorrect.

- https://github.com/matter-labs/era-system-contracts/blob/sb-boojum-integration-patch/contracts/libraries/SystemContractsCaller.sol#L42

- `META_HEAP_SIZE_OFFSET = 64`
- `META_AUX_HEAP_SIZE_OFFSET = 96`

While in `era-zkevm_opcode_defs`:
- https://github.com/matter-labs/era-zkevm_opcode_defs/blob/v1.4.1/src/definitions/abi/meta.rs#L17

`aux_heap_size` has offset 64 and `heap_size` has offset 96.

Reply:
@vladbochok are you sure?

```
result.0[1] = (self.heap_size as u64) | ((self.aux_heap_size as u64) << 32);
```

i.e.

```
result.0[1] = self.heap_size + self.aux_heap_size as u64 *2^32
```

the heap size's offset is 64 and aux heap size is 64 + 32 = 96